doc/source/ray-dashboard.rst

```diff
@@ -0,0 +1,157 @@
+Ray Dashboard
+==============
+Welcome to the Ray dashboard page! 
```

Comment:
Let's get rid of this sentence. It feels inconsistent with the tone of the rest of the documentation.